MovieTime/MovieTime/View/Movies/MTMovieDetailViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MTMovieDetailViewController.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 11/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+class MTMovieDetailViewController: MTViewController {
+
+    // MARK: Outlets
+    @IBOutlet weak var imageViewBackdrop: UIImageView!
+    @IBOutlet weak var activityIndicatorBackdrop: UIActivityIndicatorView!
+    @IBOutlet weak var imageViewPoster: UIImageView!
+
+    @IBOutlet weak var labelMovieName: UILabel!
+    @IBOutlet weak var labelMovieGenres: UILabel!
+    @IBOutlet weak var labelScore: UILabel!
+    @IBOutlet weak var labelLanguage: UILabel!
+
+    @IBOutlet weak var labelMovieDuration: UILabel!
+    @IBOutlet weak var labelReleaseDate: UILabel!
+
+    @IBOutlet weak var textViewOverview: UITextView!
+
+    // MARK: Properties
+    override class var storyboardName: String {
+        return MTIdentifiers.Storyboard.Movie.name
+    }
+
+    fileprivate var viewModel: MTMovieViewModel? {
+        return baseViewModel as? MTMovieViewModel
+    }
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return UIStatusBarStyle.lightContent
+    }
+
+    // MARK: Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupUI()
+        setupViewModel()
+    }
+
+    private func setupUI() {
```

Comment:
I tried putting a dark color with a white background and create a contrast between this two colors (black and white).
I would test other color tones and if nothing works, it is possible to put a shadow in the image corner with an alpha of 0.6 / 0.7 and then put the image on top. This would make it appear for all possible images without breaking the UI.